server/repositories/browserscope.js

```diff
@@ -2,13 +2,13 @@ const https = require('https');
 const querystring = require('querystring');
 
 exports.register = function (server, options, next) {
-  server.expose('addTest', function (title, description, slug) {
+  server.expose('addTest', function (title, description, slug, rev) {
     return new Promise(function (resolve, reject) {
       var qs = querystring.stringify({
         'api_key': options.api_key,
         name: title,
         description: description.substr(0, 60),
-        url: options.scheme + '://' + options.domain + '/' + slug
+        url: `${options.scheme}://${options.domain}/${slug}/${rev}`
```

Comment:
I originally designed the URLs this way because it felt cleaner. No big deal if this changes, though